apps/web/pages/api/auth/signup.ts

```diff
@@ -130,6 +130,15 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
       // Accept any child team invites for orgs and create a membership for the org itself
       if (team.parentId) {
+        // Create a membership for the organization itself
+        await prisma.membership.create({
+          data: {
+            userId: user.id,
+            teamId: team.parentId,
+            accepted: true,
+            role: MembershipRole.MEMBER,
+          },
+        });
```

Comment:
This was removed here https://github.com/calcom/cal.com/pull/11449/files#r1330703314